sources/platform/actors/publishing/monetize/pay_per_result.mdx

```diff
@@ -14,13 +14,13 @@ import TabItem from '@theme/TabItem';
 
 In this model, you set a price per 1,000 results. Users are charged based on the number of results your Actor produces and stores in the run's default dataset. Your profit is calculated as 80% of the revenue minus platform usage costs.
 
-The details on how your cost is computed can be found in [Example of a pay-per-result pricing model](#example-of-a-pay-per-result-pricing-model).
+The details on how your cost is computed can be found in [Example of a PPR pricing model](#example-of-a-ppr-pricing-model).
 
-## Pay-per-result (PPR) vs. pay-per-event (PPE)
+:::tip Additional benefits
```

Comment:
We should definitely have a section to encourage PPE vs PPR. The main thing is that PPR cannot charge for actor-start so it leads to more potential of profit-breaking 0 result runs

Reply:
I will solve this in follow-up PRs. I track it in this issue: https://github.com/apify/apify-docs/issues/1914.